src/windows/editor_object_selection.c

```diff
@@ -1888,7 +1903,7 @@ static rct_string_id get_ride_type_string_id(const ObjectRepositoryItem * item)
 	for (int i = 0; i < 3; i++) {
 		uint8 rideType = item->RideType[i];
 		if (rideType != 255) {
-			result = 2 + rideType;
+			result = STR_RIDE_NAME_SPIRAL_ROLLER_COASTER + rideType;
```

Comment:
Ah. This _is_ the new function (`get_ride_type_string_id`). Sorry, thought it was a duplicate while rebasing.